### PR TITLE
Ianhelle/ip explorer v2 2022 5 5

### PR DESCRIPTION
**Pull request by ianhelle**
IP Explorer notebook

### CODE REVIEW

File: Entity Explorer - IP Address.ipynb [hunk 1]
```diff
@@ -6,1891 +6,573 @@
    "source": [
```
Comment (petebryan) [inline Entity Explorer - IP Address.ipynb:6]: <p>Odd title formatting</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (petebryan) [inline Entity Explorer - IP Address.ipynb:6]: <pre class="ql-syntax" spellcheck="false">Line #1.    from pathlib import Path
</pre><p>Is Path used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (petebryan) [inline Entity Explorer - IP Address.ipynb:6]: <pre class="ql-syntax" spellcheck="false">Line #2.    from IPython.display import display, HTML
</pre><p>Is HTML used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (petebryan) [inline Entity Explorer - IP Address.ipynb:6]: <pre class="ql-syntax" spellcheck="false">Line #9.    # !pip install msticpy[azsentinel]
</pre><p>%pip install</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (petebryan) [inline Entity Explorer - IP Address.ipynb:6]: <pre class="ql-syntax" spellcheck="false">Line #14.        # additional_packages=["msticnb&gt;=0.3"],
</pre><p>Is there a reason for this to be commented out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (petebryan) [inline Entity Explorer - IP Address.ipynb:6]: <pre class="ql-syntax" spellcheck="false">Line #3.    ip_address = "147.78.47.75"
</pre><p>I think we should make the default an internal IP</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Comment by petebryan** [comment]
As with the Account one, I think the IP notebook could do with having the markdown changed to make it less of a how to use notebooklets tutorial.

**Comment by ianhelle** [comment]
I've moved a lot of the "instructional" text to appendices in both of these notebooks and provided a bit more contextual information on what is happening. I've also moved a bunch of peripheral detail (like options) into details/summary drop-down blocks.

Let me know if you think it needs more.

**Review by petebryan — APPROVED** [review]

**Pull request merged**